File: openfinalfinal/u202120418/u202120418/src/main/java/org/hign/platform/u202120418/assessment/infrastructure/persistence/jpa/repositories/MentalStateExamRepository.java
package org.hign.platform.u202120418.assessment.infrastructure.persistence.jpa.repositories;

import org.hign.platform.u202120418.assessment.domain.model.aggregates.MentalStateExam;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

/**
 * @author Fiorella Jarama Peñaloza
 * @version 1.0
 */
@Repository
public interface MentalStateExamRepository extends JpaRepository<MentalStateExam, Long> {
}

File: web/HIGN/HIGN/Assessment/Infrastructure/Persistence/EFC/Repositories/MentalStateExamRepository.cs
using HIGN.Assessment.Domain.Model.Aggregates;
using HIGN.Assessment.Domain.Repositories;
using HIGN.Shared.Infrastructure.Persistence.EFC.Configuration;
using HIGN.Shared.Infrastructure.Persistence.EFC.Repositories;

namespace HIGN.Assessment.Infrastructure.Persistence.EFC.Repositories;

public class MentalStateExamRepository(AppDbContext context): BaseRepository<MentalStateExam>(context), IMentalStateExamRepository
{
    
}